releases.go

```diff
@@ -126,6 +126,7 @@ func (s *ReleasesService) GetRelease(pid interface{}, tagName string, options ..
 type CreateReleaseOptions struct {
 	Name        *string               `url:"name" json:"name"`
 	TagName     *string               `url:"tag_name" json:"tag_name"`
+	TagMessage  *string               `url:"tag_message" json:"tag_message"`
```

Comment:
Hmm... Could you add `omitempty` for all these fields (`Name`, `tagName`, `TagMessage` and `Description`) so they are consistent and inline with the rest again? Not sure when this happened, but in option structs this should be the standard...

Reply:
Added `omitempty` onto all of them!